data/data/aws/route53/openshift.tf

```diff
@@ -3,17 +3,17 @@ locals {
   private_endpoints_count = "${var.private_endpoints ? 1 : 0}"
 }
 
-data "aws_route53_zone" "tectonic" {
+data "aws_route53_zone" "openshift" {
```

Comment:
Potential generic name here would be `base` to match `base_domain`.  I'd also be fine with `zone` or `cluster` or similar.  I'm not terribly opposed to `openshift` here though, if folks prefer that to my alternatives.